plugins/headlamp-plugin/test-plugins-examples.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# A humble test of the plugins/examples
+
+set -e
+set -o xtrace
+
+
+cd ../examples
+for i in * ; do
+  if [ -d "$i" ]; then
```

Comment:
Nit: We could remove this `if` block if we were iterating through a `find` with the type = d option. But it's okay.